src/main/java/hudson/plugins/cobertura/CoberturaPublisher.java

```diff
@@ -54,7 +54,9 @@
 
     private final String coberturaReportFile;
 
-    private final boolean onlyStable;
+    private final boolean unstable;
```

Comment:
There is no data migration code. You are going to break existing plugin installations